### ISSUE TITLE
bndrun launch configuration references non-existent project

### ISSUE THREAD

**Issue by db82407** [bndtools/bndtools#1606]
When using bndtools in a Maven workspace, trying to launch a bndrun file using "Run OSGi" sometimes fails with the following error:

"Launching xxx.bndrun has encountered a problem.
Launch configuration xxx.bndrun configuration references non-existing project x.y.z."

In order to work-around this problem, it is necessary to navigate to Run Configurations... and remove the offending bndrun entry.

**Comment by gamerson** [comment]
I too have seen this recently.  Whoever can nail down a set a reproducible steps post it here and I can take a look.

**Comment by nport** [comment]
I am facing with this issue also on Bndtools 3.5.0.REL on Eclipse Neon. Windows 10.
In my project I am using the bnd maven toolchain.

If you try to Debug once a bndrun file from the Eclipse Debug Configurations dialog. You get the reported error dialog. The bad thing is that this problem is not recoverable because if then you open the bndrun file dialog and press the "Debug OSGi" button you get the same error.
As reported by db82407 there is no way to get rid of it unless you rename the bndfile. Even if you delete the affected bndrun file and recreate it with the same name it doesn't run anymore!
It can be reproduces also on the following example (I tried with the bndrun file in branch hello-1.13.x):
    https://github.com/paremus/hello-examples

**Comment by rotty3000** [comment]
What happens if you try deleting all the eclipse run configurations
associated with that bundle or at least with the bndrun file and try
launching again?

**Comment by nport** [comment]
Good point! If I open the Run/Debug configurations dialog and I delete the configuration associated to the bndrun file, I am then able to run it again from the BndTools Resolve/Run dialog. 

When I run it from the Resolve/Run dialog the debug configuration is re-created again, but of course it doesn't work again ...

Great. You find a tip to allow to regain control of the bndrun file, but I guess that I worth to fix this issue because it prevents the use of F11 key to re-run the last launched configuration.
